pkg/gui/context/filtered_list.go

```diff
@@ -39,18 +39,18 @@ func (self *FilteredList[T]) GetFilter() string {
 	return self.filter
 }
 
-func (self *FilteredList[T]) SetFilter(filter string) {
+func (self *FilteredList[T]) SetFilter(filter string, useFuzzySearch bool) {
```

Comment:
I wonder if passing in the fuzzy find function itself rather than a boolean would make for a cleaner approach compared to passing the boolean everywhere. What do you think?

Reply:
I don't understand how that would improve it, but maybe I just don't get what exactly you are suggesting. Feel free to push a fixup.

Note that I added another fixup (81c579736c54) which needs the bool, though.

Reply:
Ah, but we could also get rid of the `shouldRetainSortOrder` stuff again, I don't need it any more since substring matching always retains the sort order, and fuzzy filtering tends to work better without.